openlibrary/core/models.py

```diff
@@ -74,7 +74,13 @@ def info(self):
             # coverstore is down
             return None
 
-    def url(self, size="M"):
+    def get_aspect_ratio(self) -> float | None:
+        info = self.info()
+        if info and "width" in info and "height" in info:
+            return info["width"] / info["height"]
```

Comment:
Seems like these can be `None`!
```suggestion
        if info and item.get('width') and item.get('height'):
            return info["width"] / info["height"]
```